utilities_common/cli.py

```diff
@@ -125,8 +125,7 @@ def __init__(self, db=None):
             self.port_dict = self.config_db.get_table('PORT')
         self.alias_max_length = 0
 
-
-        if not self.port_dict:
+        if self.port_dict is None:
```

Comment:
If the get_table API doesn't return None, this check may not be needed and we can remove it altogether. Can you please confirm?

Reply:
I agree. Both get_table() and get_port_table() initialize an empty dictionary which is being returned even if 'PORT' table is empty. I will remove this check.